src/app/QuickStarts/quickstarts/start-a-recording.tsx

```diff
@@ -18,7 +18,7 @@ import { FeatureLevel } from '@app/Shared/Services/service.types';
 import { QuickStart } from '@patternfly/quickstarts';
 import { CryostatIcon, conclusion } from '../quickstart-utils';
 
-const displayName = 'Start a Recording';
+const displayName = 'Start a recording';
```

Comment:
This one should stay the same I think

Reply:
Yes, this was one of the confusion that I discussed with you earlier and I missed it.